packages/legend-extension-dsl-persistence/src/graph/metamodel/pure/model/packageableElements/persistence/DSL_Persistence_Persistence.ts

```diff
@@ -35,7 +36,8 @@ export class Persistence
   documentation!: string;
   trigger!: Trigger;
   service!: PackageableElementReference<Service>;
-  persister!: Persister;
+  persister: Persister | undefined;
```

Comment:
Please modify this to
persister?: Persister | undefined;

Reply:
Updated.